src/test/java/com/softserveinc/edu/ita/locators/AddProductPageLocators.java

```diff
@@ -54,8 +54,11 @@ public String getName() {
         return this.name;
     }
 
-    //This method prepares locator using additional parameter by means of so called "string-format" method.
-    public AddProductPageLocators modify(String parameter) {
+    /**
```

Comment:
Rewrite this javadoc: Modifies the locator by inserting the given string
Do this for all the locator classes.